### DISCUSSION TITLE
Lanbon 4 way light switch (not 3 way)

### DISCUSSION THREAD

**Discussion by acabrele** [HASwitchPlate/openHASP#744] [Hardware]
I have a version of the lanbon L8 switch, but with 4 light relays instead of 3.  All 4 relays fire with the original firmware.

I have been trying to build a version of openhasp with the additional L4 GPIO and/or add it to the webUI configuration/GPIO section but have not had any luck.  I know that the GPIO pin is pin 15, as advised by the manufacturer.  They also confirmed 12,14 and 27 are the existing pins, which I already knew.

I have tried adding pin 15 output like the other 3 but with no luck.  I have also tried adding pin -D RELAY_4 to the lanbon build (and associate line in hasp_gpio.cpp).  But no luck.

What am I missing?

**Comment by marcoCasamento** [comment]
do you ever found a solution to this ?

**Reply by acabrele** [reply]
I did get it working yes but can't remember exactly all the changes - I could figure it out if needed.

**Reply by marcoCasamento** [reply]
well, no rush,but if you can please YES. 
It would also serve as KB.
I've got a similar plate and struggled for the exact same reason. I've tried various configuration in GPIO config UI/file with no luck so far.

**Reply by Far2Sleepy** [reply]
if you can yes please. Same situation
